docs/Simorgh-Release-Info.md

```diff
@@ -3,52 +3,52 @@ This table can then be generated using the following command: `node scripts/simo
 | cymrufyw | [articles](http://localhost:7080/cymrufyw/erthyglau/c06p32z9x2mo) |  |  |  | __Articles__: Sept 2019 |
-| gahuza | [home](http://localhost:7080/gahuza)<br/>[articles](http://localhost:7080/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](http://localhost:7080/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](http://localhost:7080/gahuza/amakuru-23257470)<br/>[PGL](http://localhost:7080/gahuza/amakuru-43894701)<br/>[STY](http://localhost:7080/gahuza/23299713) | [home](https://www.test.bbc.com/gahuza)<br/>[articles](https://www.test.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.test.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.test.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.test.bbc.com/gahuza/23111981)<br/>[STY](https://www.test.bbc.com/gahuza/23204135) | [home](https://www.stage.bbc.com/gahuza)<br/>[articles](https://www.stage.bbc.com/gahuza/articles/cey23zx8wx8o)<br/>[liveRadio](https://www.stage.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.stage.bbc.com/gahuza/amakuru-23257470)<br/>[PGL](https://www.stage.bbc.com/gahuza/23111981)<br/>[STY](https://www.stage.bbc.com/gahuza/23204135) | [home](https://www.bbc.com/gahuza)<br/>[articles](https://www.bbc.com/gahuza/articles/cryd02nzn81o)<br/>[liveRadio](https://www.bbc.com/gahuza/bbc_gahuza_radio/liveradio)<br/>[MAP](https://www.bbc.com/gahuza/amakuru-51710168)<br/>[PGL](https://www.bbc.com/gahuza/amakuru-43894701)<br/>[STY](https://www.bbc.com/gahuza/amakuru-52821373) | __Home__: 19th Feb 2020<br/>__Articles__: Sept 2019<br/>__Live Radio__: 10th Jan 2020<br/>__MAP__: 27th Feb 2020<br/>__PGL__: 12th March 2020<br/>__STY__: 28th May 2020 |
```

Comment:
`[STY](https://www.bbc.com/gahuza/amakuru-52821373)`

I think you removed the LIVE link.

Reply:
oh yeah, let me check the cypress settings for STY 👁️ 👁️